docs/data/registry.json

```
@@ -312,11 +312,9 @@
     "versions": {
       "deprecated": [],
       "supported": [
+        "19"
```

:clipboard: The build breaks because of this missing comma
```suggestion
        "19",
```